packages/suite/src/components/suite/modals/ReduxModal/CopyAddressModal.tsx

```diff
@@ -47,7 +60,14 @@ export const CopyAddressModal = ({ address, onCancel, addressType }: CopyAddress
             icon="warningTriangleLight"
             iconVariant="warning"
             bodyHeading={<Translation id="TR_NOT_YOUR_RECEIVE_ADDRRESS" />}
-            text={<Translation id={getAddressTypeText(addressType)} />}
+            body={
+                <>
+                    <Translation id={getAddressTypeText(addressType)} />
+                    <StyledCheckbox isChecked={checked} onClick={() => setChecked(!checked)}>
```

Comment:
You don't need to override styles here (we try to avoid this pattern):

```tsx
<Checkbox isChecked={checked} onClick={() => setChecked(!checked)} margin={{ top: spacings.md }}>
```

Reply:
Thanks for letting me know. Will do it like this in the future.

Reply:
Thanks 🙏 

In the future this pattern should be available for all related design system components